### DISCUSSION TITLE
Use Makefile to centralize scripts

### DISCUSSION THREAD

**Discussion by PaulRBerg** [paulrberg/foundry-template#1] [Ideas]
Users currently need to run some commands with `forge`, other commands with `yarn`, and other commands with `anvil`.

**Comment by ahmedali8** [comment]
Hi @PaulRBerg, I think so it's a great idea to have a Makefile for commands such as  in [this](https://twitter.com/PaulRBerg/status/1630911941655904258?s=20) tweet of yours.

**Reply by PaulRBerg** [reply]
Maybe .. Since I opened this discussion, I realized that having to maintain both a `package.json` file and a `Makefile` might be overkill for this template.

I'll have a think.

**Reply by ahmedali8** [reply]
maybe we could only have a Makefile to handle all the package.json scripts as well to centralize commands

**Reply by PaulRBerg** [reply]
Interesting idea.
